Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/IndexService.java

```diff
@@ -6,13 +6,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
not sure what you mean, but do you see that the lines do not all start at the same location?

cmd-f should format your code according to the style you've defined.